### PR TITLE
Added 'versionadded' tags to sensehat modules

### PR DESCRIPTION
**Pull request by benediktwerner**

### CODE REVIEW

File: salt/beacons/sensehat.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 Beacon to monitor temperature, humidity and pressure using the SenseHat
 of a Raspberry Pi.
 
+.. versionadded:: 2017.7
```
Comment (rallytime) [inline salt/beacons/sensehat.py]: `2017.7` --> `2017.7.0`

**Review by rallytime — CHANGES_REQUESTED** [review]
Hi @benediktwerner - thanks for adding these version directives.

Can you update both of those references from `2017.7` to `2017.7.0`? That's just a little more specific to the release version sensehat support was added.

**Comment by benediktwerner** [comment]
Sure, done.

**Review by rallytime — APPROVED** [review]
Thanks!

**Pull request merged**